### DISCUSSION TITLE
Disable the layout pop-up on switching spaces

### DISCUSSION THREAD

**Discussion by cueo** [ianyh/Amethyst#1259] [Q&A] [answered]
Is it possible to disable the pop-up (is it called HUD?) that shows the current layout when switching spaces?

**Comment by ianyh** [comment]
Yes. There's an option in preferences to disable it on switching spaces.

**Comment by sleepymalc** [comment, marked as answer]
See the following.
<img width="584" alt="image" src="https://user-images.githubusercontent.com/60427058/178867345-3a2285f3-4190-4d5a-9b9a-22f3f800ed0e.png">

**Reply by sleepymalc** [reply]
In this case, HUD will be triggered only when I explicitly switch the layout, and will not be triggered when I switch spaces.
